service-workers/service-worker/navigation-preload/navigationpreload.https.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>ServiceWorker: navigator.serviceWorker.navigationPreload</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../resources/test-helpers.sub.js"></script>
+<script src="resources/helpers.js"></script>
+<script>
+promise_test(async t => {
+  const SCRIPT = '../resources/empty-worker.js';
+  const SCOPE = '../resources/navigationpreload';
+  const registration =
+      await service_worker_unregister_and_register(t, SCRIPT, SCOPE);
+  const np = registration.navigationPreload;
+  assert_true(np instanceof NavigationPreloadManager, 'navigationPreload ' +
+              'attribute return NavigationPreloadManager object');
```

Comment:
return -> should return

Reply:
Updated, please review my latest commit, thanks a lot.